packages/dd-trace/src/appsec/iast/taint-tracking/index.js

```diff
@@ -10,18 +10,28 @@ const {
 } = require('./operations')
 
 const taintTrackingPlugin = require('./plugin')
+const kafkaConsumerPlugin = require('./plugins/kafka')
+
+const kafkaContextPlugin = require('../context/kafka-ctx-plugin')
 
 module.exports = {
   enableTaintTracking (config, telemetryVerbosity) {
     enableRewriter(telemetryVerbosity)
     enableTaintOperations(telemetryVerbosity)
     taintTrackingPlugin.enable()
+
+    kafkaContextPlugin.enable()
```

Comment:
Perhaps context plugins should be initialized in an specialized module by the iast index module before enabling taint tracking. I guess some day there will be more `IastContextPlugin`s than kafka plugin...